python/geometryApproximation.py

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env python
+
+from paraview.simple import *
+
+# create a new 'XML PolyData Reader'
+streamlinesvtp = XMLPolyDataReader(FileName=['/home/emma/ttk/ttk-source/ttk-data-emmwel/GroundWater.cdb/streamlines.vtp'])

```

Comment:
I don't believe the streamlines need to be loaded in this srcipt.
(they are not used further down the script)